client/src/components/createVote/CreateVotePage.tsx

```diff
@@ -99,18 +120,56 @@ export default class CreateVotePage extends Component<ICreateVotePageProps, ICre
   };
 
   public render() {
-    if (this.state.mode === "form") {
-      return (
-        <CreateVoteForm
-          setSubmitData={this.setSubmitData}
-          formData={this.state.formData}
-          blockchainData={this.props.blockchainData}
-        />
-      );
-    } else if (this.state.mode === "fetching") {
-      return <LoadingResult getTransactionResult={this.getTransactionResult} />;
-    } else {
-      return <DisplayResult status={this.state.resultStatus} onClick={this.setModeToForm} />;
-    }
+    return (
+      <Grid>
+        <Row>
+          {this.state.mode === PageMode.Form ? (
+            <Col md={6} mdOffset={3}>
```

Comment:
I think we could stretch it a little.

```suggestion
            <Col md={10} mdOffset={1}>
```